.github/workflows/review-requirements/update-all-prs.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  schedule:
+    # Every 10 minutes
+    cron: "*/10 * * * *"
```

Comment:
Is this a bit aggressive given the volume we actually deal with? Maybe every hour (or even two) would be enough, given it's unlikely we'll have a PR waiting to merge the very moment it's _technically_ had enough time.

Reply:
Yeah. I'll shift to every hour. Could probably also limit it to working hours (if I can work out the `cron` expression for that).

Reply:
Done.